Core/Core/Views/CoreWebView.swift

```diff
@@ -273,3 +279,35 @@ extension CoreWebView {
         }
     }
 }
+
+extension CoreWebView {
+    private func replaceContentViewClass() {
```

Comment:
What is this method doing exactly?  What are we replacing it with?  Is this specific to RCE?  Should this live closer to RCE stuff?  Does this make our general purpose CoreWebView RCE specific?

Reply:
This is handling inputAccessoryView, which I'd imagine isn't only for RCE, but that's the only use so far.

Reply:
Would `replaceContentInputAccessoryView` be a better name? I think something to indicate that it's swizzling the input accessory view would be less confusing.